SymmetricMatrixConstantCoefficient::Eval is ambiguous if DenseMatrix is the first argument

I am reporting an error which I saw on Github MacOS runner. This is from build log of PyMFEM and bit messy. But, what it is saying is
that there are two candidate for ``` (arg1)->Eval(*arg2,*arg3,(mfem::IntegrationPoint const &)*arg4)``` . 
Here arg1 is ```SymmetrixMatrixConstantCoefficient``` abd  args2, arg3, arg4 are ```DenseMatrix```, ```ElementTrandformation```, and ```IntegrationPoint```, respectively. 

Apparently, the compiler thinks there are two candidates. One is coefficient.hpp:945, which is MatrixCoefficient::Eval and
coefficient.hpp:1331, which is SymmetricMatrixCoefficient::Eval. The first one is defined as =0, it looks strange to my
eyes, but it is also true that there is using MatrixCoefficient::Eval in line 1351. 

```
2023-01-04T20:22:52.3787910Z coefficient_wrap.cxx:19522:15: error: call to member function 'Eval' is ambiguous
2023-01-04T20:22:52.3789130Z       (arg1)->Eval(*arg2,*arg3,(mfem::IntegrationPoint const &)*arg4);
2023-01-04T20:22:52.3789850Z       ~~~~~~~~^~~~
2023-01-04T20:22:52.3790920Z /Users/runner/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/mfem/external/ser/include/mfem/fem/coefficient.hpp:945:17: note: candidate function
2023-01-04T20:22:52.3792350Z    virtual void Eval(DenseMatrix &K, ElementTransformation &T,
2023-01-04T20:22:52.3792960Z                 ^
2023-01-04T20:22:52.3795440Z /Users/runner/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/mfem/external/ser/include/mfem/fem/coefficient.hpp:1331:17: note: candidate function
2023-01-04T20:22:52.3796340Z    virtual void Eval(DenseMatrix &K, ElementTransformation &T,
2023-01-04T20:22:52.3796930Z                 ^
```

--- a/fem/coefficient.hpp
+++ b/fem/coefficient.hpp
@@ -1347,7 +1347,6 @@ public:
    ///Construct using matrix @a m for the constant.
    SymmetricMatrixConstantCoefficient(const DenseSymmetricMatrix &m)
       : SymmetricMatrixCoefficient(m.Height()), mat(m) { }
-   using MatrixCoefficient::Eval;
    using SymmetricMatrixCoefficient::Eval;
    /// Evaluate the matrix coefficient at @a ip.
    virtual void Eval(DenseSymmetricMatrix &M, ElementTransformation &T,

--- a/fem/coefficient.hpp
+++ b/fem/coefficient.hpp
@@ -1319,7 +1319,6 @@ public:
    virtual void Eval(DenseSymmetricMatrix &K, ElementTransformation &T,
                      const IntegrationPoint &ip) = 0;
 
-   using MatrixCoefficient::Eval;
    /** @brief Evaluate the matrix coefficient in the element described by @a T
        at the point @a ip, storing the result as a dense matrix @a K. */
    /** This function allows the use of SymmetricMatrixCoefficient in situations
@@ -1400,7 +1399,6 @@ public:
    /// Set the time for internally stored coefficients
    void SetTime(double t);
 
-   using MatrixCoefficient::Eval;
    using SymmetricMatrixCoefficient::Eval;
    /// Evaluate the matrix coefficient at @a ip.
    virtual void Eval(DenseSymmetricMatrix &K, ElementTransformation &T,

